doc/SSH.md

```diff
@@ -0,0 +1,71 @@
+# SSH Storage Engine Configuration
+
+## Prerequisites
+First of all, you need to have a remote server where you can store your backups on.
+
```

Comment:
Lets do

How it works -- backup locally then transfer "remote"

* Prerequisites
* Local / remote sshd
* EC2